docs/dev.rst

```diff
@@ -237,18 +208,17 @@ To check which Python packages are outdated, use `piprot`_ in a shell:
 .. code-block:: shell
 
    $ make shell
-   > pip install piprot
-   > piprot -o
+   root@...:/app# pip install piprot
+   root@...:/app# piprot -o requirements.txt
 
 The ``-o`` flag means it only lists requirements that are *out of date*.
 
 .. note:: A good idea is to install ``hashin`` and ``piprot`` globally
    on your computer instead. It doesn't require a virtual environment if
-   you use `pipsi`_.
+   you use `pipx`_.
 
 .. _piprot: https://github.com/sesh/piprot
-.. _pipsi: https://github.com/mitsuhiko/pipsi
-
+.. _pipx: https://pypi.org/project/pipx/
```

Comment:
pipx forked pipsi because pipsi wasn't being maintained. Worth updating things to talk about pipx now.

Reply:
I had no idea! Glad I skimmed this PR. 
You know, I never got it into my muscle memory to use pipsi because my mac's broken Python global installs. I can't remember what happened but what "python" meant changed back and forth (today it means 2.7) and it totally screwed up my pipsi.

Reply:
Yeah, I hear you on that. I started using pyenv to manage python environments/versions. That's made it a lot easier to work on libraries and other things. Have you tried it?

https://github.com/pyenv/pyenv